src/components/MenuBar/MenuBarToggleButton.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+    IconButton,
+    IconButtonProps,
+    Tooltip,
+    TooltipTypeMap,
+} from "@mui/joy";
+
+import Check from "@mui/icons-material/Check";
+import Close from "@mui/icons-material/Close";
+
+
+interface MenuBarToggleButtonProps extends IconButtonProps {
+    tooltipPlacement?: TooltipTypeMap["props"]["placement"];
+    isActive?: boolean;
+    onIcon?: React.ReactNode;
+    offIcon?: React.ReactNode;
+}
+
+/**
+ * A toggle button for use in the menu bar.
+ *
+ * @param props
+ * @param props.title Tooltip title.
+ * @param props.tooltipPlacement Tooltip position.
+ * @param props.onIcon Icon when active.
+ * @param props.offIcon Icon when inactive.
+ * @param props.rest Other IconButton props.
+ * @param props.isActive
+ * @return JSX.Element
+ */
+const MenuBarToggleButton = ({
+    tooltipPlacement,
+    title,
```

Comment:
How about
```suggestion
    tooltipTitle,
```
since `title` is usually used to set a native HTML tooltip.

Then let's also add this to MenuBarToggleButtonProps.

Reply:
Fixed! I also fixed the `MenuBarIconButton.tsx` because it has the same issue.